src/svc_vc.c

```diff
@@ -151,12 +151,6 @@ svc_vc_ncreate2(int fd, u_int sendsize, u_int recvsize, u_int flags)
 	/* attach shared state */
 	if ((oflags & RPC_DPLX_LKP_OFLAG_ALLOC) || (!rec->hdl.xd)) {
 		xd = rec->hdl.xd = alloc_x_vc_data();
```

Comment:
Actually, I should know about them -- as I wrote them some time ago. Anyway, I've added another patch in the series with the asserts.